gateway-ha/src/test/java/io/trino/gateway/ha/HaGatewayTestUtils.java

```diff
@@ -174,12 +170,11 @@ private static void verifyTrinoStatus(int port, String name)
         throw new IllegalStateException("Trino cluster is not healthy");
     }
 
-    public record TestConfig(String configFilePath, String h2DbFilePath)
+    public record TestConfig(String configFilePath)
```

Comment:
This record now contains a single field. If you rename `buildGatewayConfig` to `getConfigPath` and have it return a `String` it should simplify the code while still being clear.

Reply:
`get` prefix looks wrong because the method has other operations like writing files.

Reply:
True. Lets just return `target` as a `File` and continue using `buildGatewayConfig` as the method name.